<launch>



    <group ns="robot1">

        <include file="$(find cpp)/launch/for_multi_dev_rtabmap.launch">
            <arg name="for_frame" value="robot1/" />
            <arg name="topic_name" value="/robot1" />
        </include>
    </group>


    <group ns="robot2">

        <include file="$(find cpp)/launch/for_multi_dev_rtabmap.launch">
            <arg name="for_frame" value="robot2/" />
            <arg name="topic_name" value="/robot2" />
        </include>
    </group>

    <!-- <node pkg="cpp" name="image_extractor" type="image_extractor.py" output="screen"/>
    <node pkg="cpp" name="feature_matchor" type="feature_matching.py" output="screen"/>
    <node pkg="cpp" name="loop_closure_server" type="detection_checker" output="screen"/>
    <node pkg="cpp" name="RO_node" type="RO_estimator" output="screen"/> -->
    <!-- <node pkg="cpp" name="open3d_conversions_ex_sub" type="open3d_conversions_ex_sub" output="screen"/> -->


</launch>